src/config/entities/__tests__/configuration.ts

```diff
@@ -84,20 +84,32 @@ export default (): ReturnType<typeof configuration> => ({
       apiKey: faker.string.hexadecimal({ length: 32 }),
     },
   },
-  typeorm: { autoLoadEntities: true, manualInitialization: true },
   db: {
-    postgres: {
-      schema: process.env.POSTGRES_SCHEMA || 'main',
-      host: process.env.POSTGRES_TEST_HOST || 'localhost',
-      port: process.env.POSTGRES_TEST_PORT || '5433',
-      database: process.env.POSTGRES_TEST_DB || 'test-db',
-      username: process.env.POSTGRES_TEST_USER || 'postgres',
-      password: process.env.POSTGRES_TEST_PASSWORD || 'postgres',
-      ssl: {
-        enabled: true,
-        requestCert: true,
-        rejectUnauthorized: true,
-        caPath: process.env.POSTGRES_SSL_CA_PATH,
+    migrator: {
+      migrationsExecute: true,
+      numberOfRetries: process.env.DB_TEST_MIGRATIONS_NUMBER_OF_RETRIES ?? 5,
+      retryAfter: process.env.DB_TEST_MIGRATIONS_RETRY_AFTER ?? 1000, // Milliseconds
+    },
+    orm: {
+      autoLoadEntities: true,
+      manualInitialization: true,
+      migrationsRun: false,
+      migrationsTableName: 'migrations_typeorm',
```

Comment:
Nit: what do you think about prefixing this with an underscore? It's highly unlikely we'll move to another ORM but I suggest we remove `_typeorm` to be more generic as we are with all datasources.

```suggestion
      migrationsTableName: '_migrations_typeorm',
```

Reply:
Well the idea was to make it clear who owns the table. Will change the name.

Reply:
We can keep the name if you think it's better - it was a suggestion based on the thought that there _may_ be migration in the future.

Reply:
Changed the name
https://github.com/safe-global/safe-client-gateway/commit/2ea0e5d8a752fd31cff745ff3490d8fe8134ddc7